.ignore

```diff
@@ -1,2 +1,3 @@
 # Ignores for rg or ag
 patches/
+packages/dashboard/src/assets/images/SetupTutorial.svg
```

Comment:
why?

Reply:
It's a generated animate svg that contains mass of generated code. No need to scan over it.

Reply:
Any code in this repo generates it? If not, we should keep it tracking still. Otherwise, we should delete it from tracking and generate it from CI.

Reply:
It's not ignore for git, just for [ripgrep](https://github.com/BurntSushi/ripgrep) or [ag](https://github.com/ggreer/the_silver_searcher)

Reply:
> Any code in this repo generates it?

No. It's exported by design tooling.